<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>686</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1058</width>
    <height>686</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>30000</width>
    <height>30000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DIR1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../Downloads/icono1.jpg</normaloff>../../../../../../../Downloads/icono1.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background: rgb(123, 158, 255);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QFrame" name="frameIzq">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="titGrafLong">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Gr√°fico de corte Longitudinal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="titGrafTrans">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>5000</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Grafico de corte Transversal por cada cota</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QFrame" name="frameDer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>2000</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::WrapLongRows</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
            <property name="horizontalSpacing">
             <number>20</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="btnUpload">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-radius: 15px;
	background: rgb(0,85,255);
	color: white;
}
QPushButton:hover{
	background: rgb(52, 116, 255);
}
QPushButton:pressed{
	background: rgb(70, 184, 255);
}</string>
              </property>
              <property name="text">
               <string>Cargar Excel</string>
              </property>
              <property name="icon">
               <iconset theme="accessories-calculator"/>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="btnSave">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-radius: 15px;
	background: rgb(0,85,255);
	color: white;
}
QPushButton:hover{
	background: rgb(52, 116, 255);
}
QPushButton:pressed{
	background: rgb(70, 184, 255);
}</string>
              </property>
              <property name="text">
               <string>Guardar Informe</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnConfig">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-radius: 15px;
	background: rgb(0,85,255);
	color: white;
}
QPushButton:hover{
	background: rgb(52, 116, 255);
}
QPushButton:pressed{
	background: rgb(70, 184, 255);
}</string>
              </property>
              <property name="text">
               <string>Obras de Arte</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="btnEstructural">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-radius: 15px;
	background: rgb(0,85,255);
	color: white;
}
QPushButton:hover{
	background: rgb(52, 116, 255);
}
QPushButton:pressed{
	background: rgb(70, 184, 255);
}</string>
              </property>
              <property name="text">
               <string>Paquete Estructural</string>
              </property>
              <property name="icon">
               <iconset theme="accessories-calculator"/>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="dataGrafTrans">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>[Progresiva N√∫mero: 0]
[Cota N√∫mero: 0]
[Altura de Ruta: 45m]
[Altura de Terreno Natural: 50m]
[Diferencia: 5m] </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dataGeneral">
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Dato1: √Årea de tierra retirada por cada corte seleccionado
Dato2: √Årea de tierra cargada por cada corte seleccionado
Dato3: Balance Superficie por cada corte seleccionado

Dato4: Volumen de tierra retirada total
Dato5: Volumen de tierra cargada total
Dato6: Balance total de Volumen remanente.
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dataInfo">
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::WinPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>[Repositorio de GitHub: ]
[Autores:  ]
LinkManual 
Versi√≥n: 0.2 rev5</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="titulo">
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Dise√±o Interactivo de Razante</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
